generators/markup/files/page.js

```diff
@@ -0,0 +1,61 @@
+
+import { createElement, PropTypes, Component } from 'react';
+
+function script({ path, content, type, id }, i) {
+  return path ?
+    <script type={type} src={path} key={i}/> :
+    <script type={type} key={i} id={id} dangerouslySetInnerHTML={{
+      __html: content
+    }}/>;
+}
+
+function style({ path, type, content }, i) {
```

Comment:
Was hoping to have them in a separate node module; is this something an implementor would ever want to customize? Since more things that get put in means more things to test for them and whatnot.